apps/central/src/components/admin/ManagePrivacyPolicy.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div>
+    <h3>Privacy policy</h3>
+    <div>
+      <button
```

Comment:
I would expect that you use a checkbox or select input here? Curious why you make this choice

Reply:
If it was a checkbox list or select we'd need to track whether the text field is dirty, so we can deselect that chosen option. We chose buttons because it was simpler to implement for now (and since there was still some back and forthing over the story). 
Of course this is still a draft PR so if we'd want it differently we can change it, but we need to know what exactly we want for it ;)